ui/src/app/styles/buttons.css

```diff
@@ -16,19 +16,21 @@ button.btn.btn-primary {
   line-height: 0.77rem;
 }
 
-button.btn.btn-primary:hover {
+button.btn.btn-primary:hover, a.btn.btn-primary:hover {
   background-color: #4356A7;
   color: #fff;
 }
 
-button.btn.btn-primary:disabled {
+button.btn.btn-primary:disabled, a.btn.btn-primary:disabled {
   background-color: #c3c3c3;
   color: #fff;
   cursor: auto;
 }
 
 button.btn.btn-secondary,
-button.btn.btn-outline {
+button.btn.btn-outline,
+a.btn.btn-secondary,
```

Comment:
Do we need the element selector for these? Would `btn` suffice (thereby eliminating some of the duplication)?

Reply:
Let me check. I _believe_ when I did this originally, it was to provide more specificity than the clarity styles, to override those styles.

Reply:
Appears to work without the element selector! :)